ts/packages/cli/src/commands/find.ts

```diff
@@ -0,0 +1,175 @@
+    });
+    await once(readLine, 'close');
+    return result;
+  }
+
+  private async find() {
+    const projectPath = path.resolve(this.args.path);
+    const unusedOnly = this.flags.unused;
+    const showTemplateKeys = this.flags['show-templates'];
+    await this.updateIgnoredFiles(projectPath);
+    const cfguPaths = this.flags.cfgu ? [this.flags.cfgu] : this.findCfgu(projectPath);
+
+    if (cfguPaths.length === 0)
+      throw new Error(
+        `no cfgu file found or provided. If you have another type of configuration file, run configu init --import=<CONFIG-FILE> --defaults to generate a cfgu from it`,
+      );
+
+    const keysFromCfgu = await Promise.all(
+      cfguPaths.map(async (cfguFile) => {
+        try {
+          const schemaContents = await ConfigSchema.parse(new ConfigSchema(cfguFile));
+          if (!showTemplateKeys) {
+            const schemaEntries = Object.entries(schemaContents);
+            const ignoredKeys = new Set(
+              schemaEntries
+                .filter(([key, cfgu]) => !!cfgu.template)
+                .flatMap(([key, cfgu]) => {
+                  return TMPL.parse(cfgu.template as string)
+                    .filter((templateSpan) => templateSpan.type === 'name')
+                    .map((templateSpan) => templateSpan.key);
+                }),
+            );
+            return schemaEntries.filter(([key, cfgu]) => !ignoredKeys.has(key)).map(([key, cfgu]) => key);
+          }
+          return Object.keys(schemaContents);
+        } catch {
+          return [];
+        }
+      }),
+    );
+
+    const keysRegEx = _.mapValues(_.keyBy([...new Set(keysFromCfgu.flat())]), (key) => ({
+      pattern: new RegExp(key, 'g'),
+      count: 0,
+    }));
+
+    const files = FastGlob.sync([path.join(projectPath, '/**/*.*')], {
+      dot: true,
+      ignore: [...[...this.ignoredFiles].map((dirName) => path.join(projectPath, `/**/${dirName}`)), '/**/*.json'],
+    });
+
+    await Promise.all(
+      files.map((file) =>
+        this.processLineByLine(file, (line, lineIndex) => {
+          Object.keys(keysRegEx).map(async (key) => {
+            const keyRegEx = keysRegEx[key];
+            if (keyRegEx) {
+              const regEx = keyRegEx.pattern;
+              const match = regEx.exec(line);
+              if (line && match) {
+                keyRegEx.count += 1;
+                if (!unusedOnly)
+                  this.log(`${file.replace(projectPath, '...')}:${lineIndex}:${match.index} [${regEx.source}]`);
+              }
+            }
+          });
+        }),
+      ),
+    );
+    const unusedKeys = Object.keys(keysRegEx).filter((key) => keysRegEx[key]?.count === 0);
+    if (unusedKeys.length > 0) this.log(`Unused configs found: ${unusedKeys.join(', ')}`);
+    else if (unusedOnly) this.log('No unused configs found');
+  }
+
+  public async run(): Promise<void> {
+    try {
+      await this.find();
+    } catch (error) {
+      this.log(`ERROR: ${error.message}`);
+    }
```

Comment:
Couple of suggestion here:
- find is too general name
- The log ERROR .. in the end is not really necessary 
  - we actually want to exit with code 1 when something went wrong
  - throwing an error actually logs an "Error" prefix as in the other commands
- Im suggesting moving all the code from find method to the run method without wrapping it in try catch

Reply:
sure